### PR TITLE
Fix detection of xgamma

### LINKED ISSUE #1010: “Restore default gamma....” doesn’t restore gamma after game quits
For some reason, with "Restore default gamma..." option enabled, default screen gamma is not restored after a game that changes it quits (even after Lutris kills processes in the prefix)
When running Lutris in debug mode (-d) I get this Warning:
"WARNING  2018-08-08 18:46:47,323 [display.restore_gamma:168]:xgamma is not available on your system"
Would be nice to fix this, as changing screen gamma back manually is a bit annoying.

### PR DESCRIPTION
**Pull request by luthub**
Fixes #1010

### CODE REVIEW

**Comment by CitrusAlex** [comment]
Will gamma restoration run every time any game leaves or only with 'Use XVidMode to change resolutions" (which allows games to change screen gamma) enabled?
Because if it's the former it will reset Night Light mode, which will be extremely annoying if it will happen with every game, even if it doesn't change gamma.

File: lutris/util/display.py [hunk 1]
```diff
@@ -159,11 +158,9 @@ def change_resolution(resolution):
 
 def restore_gamma():
     """Restores gamma to a normal level."""
-    global XGAMMA_FOUND
-    if XGAMMA_FOUND is None:
-        XGAMMA_FOUND = system.find_executable('xgamma')
-    if XGAMMA_FOUND is True:
-        subprocess.Popen(["xgamma", "-gamma", "1.0"])
+    XGAMMA_PATH = system.find_executable('xgamma')
```
Comment (strycore) [inline lutris/util/display.py]: Cool but this removes caching so we make a call to `which xgamma` every single time. And yes, this is way more costly than one would think.

Comment (luthub) [reply lutris/util/display.py]: You're right. I hope it's better now.

**Comment by luthub** [comment]
This PR is rubbish. I opened a new one that hopefully does it right. Sorry for the inconvenience

**Pull request closed without merging**